packages/api/src/hooks/useDynamicLeverage.ts

```diff
@@ -0,0 +1,124 @@
+// import { useMemo } from 'react';
+// import useQuery from '../useQuery';
+
+// hardcoded dynamic leverage values, to be replaced with API call
+const dynamic_leverages = [
```

Comment:
I get that its temporary... is there a way to at least track tech debt like that if its necessary to be added? cc @markwylde-deriv

Reply:
Added a TODO here, will also raise the point to create a click up card for API integration which can be picked up when endpoint is ready.

Reply:
Let's put this ins a config file somewhere, maybe `packages/api/src/config/dynamicLeverages.ts`

Reply:
This is just temporarily here until the endpoint is created, I think moving it to a config file would make it seem it should be there and we might lose track of it.